### PR TITLE
Changes in Account Requirments

### PR DESCRIPTION
**Pull request by joinimran**
As user has reported and I have tested, we need to have a Work/School account required.

Problem:  https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2424

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/application-management/msix-app-packaging-tool.md [hunk 1]
```diff
@@ -30,11 +30,11 @@ You can either run your installer interactively (through the UI) or create a pac
 
 - Windows 10, version 1809 (or later)
 - Participation in the Windows Insider Program (if you're using an Insider build)
-- A valid Microsoft account (MSA) alias to access the app from the Microsoft Store 
+- A valid Microsoft Work/School account to access the app from the Microsoft Store
```
Comment (JohanFreelancer9) [inline windows/application-management/msix-app-packaging-tool.md:33]:
```suggestion
- A valid Microsoft work or school account to access the app from the Microsoft Store 
```

File: windows/application-management/msix-app-packaging-tool.md [hunk 2]
```diff
@@ -30,11 +30,11 @@ You can either run your installer interactively (through the UI) or create a pac
...
 - Windows 10, version 1809 (or later)
 - Participation in the Windows Insider Program (if you're using an Insider build)
-- A valid Microsoft account (MSA) alias to access the app from the Microsoft Store 
+- A valid Microsoft Work/School account to access the app from the Microsoft Store 
 - Admin privileges on your PC account 
 
 ### Get the app from the Microsoft Store
 
-1. Use the MSA login associated with your Windows Insider Program credentials in the [Microsoft Store](https://www.microsoft.com/store/r/9N5LW3JBCXKF). 
+1. Use the Work/School login associated with your Windows Insider Program credentials in the [Microsoft Store](https://www.microsoft.com/store/r/9N5LW3JBCXKF).
```
Comment (JohanFreelancer9) [inline windows/application-management/msix-app-packaging-tool.md:38]:
```suggestion
1. Use the work or school account login associated with your Windows Insider Program credentials in the [Microsoft Store](https://www.microsoft.com/store/r/9N5LW3JBCXKF). 
```

**Comment by joinimran** [comment]
Suggestions applied.

**Comment by e0i** [comment]
@traya1 Is there any closure comment to share?

**Comment by traya1** [comment]
Hi @e0i,

Unfortunately, I cannot recall why I closed this specific PR. Are you able to reopen the PR? The option to reopen is disabled for me in this PR.

**Comment by e0i** [comment]
It seems that Collaborator access does not grant PR re-open rights. We'll remake this one. Thank you.

**Comment by illfated** [comment]
@e0i , @traya1 : I believe this is one of several PRs closed lately due to wrong target branch (master) for this repository. The master branch has recently become unavailable (hidden or removed) to most users.

**Pull request closed without merging**